script/Arcueid/Day 4/04_00_ARC04_6_2D.txt

```diff
@@ -729,57 +774,59 @@ In other words, although I am no longer immortal, Shiki, would you be able to cu
 [sha:3e5ae5656f27c10b314425c66dbc8f435d5bfd7f]{
 -- Page 619, Offset 7229.
 -- ……どうだろう。
-...I wonder.
+I'm not so sure...
 }
 [sha:61fc0bb5dedb3b563378038fae7f5dc704ec26e5]{
 -- Page 619, Offset 7230.
 -- たしかに線があるんだから切れるとは思う。
-It's true that I can cut anything if a line exists.
+It's true that I can cut anything that has a line through it.
```

Comment:
maybe line "on it"? "through it" is a little weird to describe something on the surface